node/main.go

```diff
@@ -35,6 +35,7 @@ func initKit() {
 		kit.ThroughRPC(),
 		kit.WithCfgOpt(func(cfg *config.FullNode) error {
 			cfg.ActorEvent.EnableRealTimeFilterAPI = true
```

Comment:
This option is now gone. It is now a negative (Disable...) and lives under the Fevm group. You can remove this line.